datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedPartitionAssignmentStrategy.java

```diff
@@ -99,8 +99,8 @@ public LoadBasedPartitionAssignmentStrategy(PartitionThroughputProvider throughp
 
     // Doing assignment
     LoadBasedPartitionAssigner partitionAssigner = new LoadBasedPartitionAssigner();
-    return partitionAssigner.assignPartitions(clusterThroughputInfo, assignedPartitions, unassignedPartitions,
-        maxTaskCount);
+    return partitionAssigner.assignPartitions(clusterThroughputInfo, currentAssignment,
```

Comment:
add validation check before sending the results to Coordinator.

Reply:
Please ignore `LoadBasedAssignmentStrategy.java` in this PR. A separate one will be published for it.